plugins/available/rbenv.plugin.bash

```diff
@@ -3,9 +3,7 @@
 cite about-plugin
 about-plugin 'load rbenv, if you are using it'
 
-pathmunge "${HOME}/.rbenv/bin"
+pathmunge $HOME/.rbenv/bin
```

Comment:
Any specific reason why you removed the double quotes? Is this still going to work in case the `$HOME` value contains spaces?